lib/db/connection.rb

```diff
@@ -18,7 +18,13 @@ def environment
     end
 
     def config
-      @config ||= DB::Config.new(environment).options
+      if environment == 'development'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.